tools/walletextension/wallet_extension.go

```diff
@@ -33,18 +27,15 @@ const (
 	PathSubmitViewingKey   = "/submitviewingkey/"
 	staticDir              = "static"
 
-	reqJSONKeyMethod          = "method"
-	reqJSONKeyParams          = "params"
-	reqJSONKeyFrom            = "from"
-	ReqJSONMethodGetBalance   = "eth_getBalance"
-	ReqJSONMethodCall         = "eth_call"
-	ReqJSONMethodGetTxReceipt = "eth_getTransactionReceipt"
-	ReqJSONMethodSendRawTx    = "eth_sendRawTransaction"
-	ReqJSONMethodGetTxByHash  = "eth_getTransactionByHash"
-	respJSONKeyErr            = "error"
-	respJSONKeyMsg            = "message"
-	RespJSONKeyResult         = "result"
-	httpCodeErr               = 500
+	reqJSONKeyID             = "id"
+	reqJSONKeyMethod         = "method"
+	reqJSONKeyParams         = "params"
+	ReqJSONMethodCall        = "eth_call"
```

Comment:
@BedrockSquirrel This and the constant below can be moved over to the integration test file (not used here).